modules/custom/az_paragraphs/css/az_paragraphs_az_text_media.css

```
@@ -38,3 +41,43 @@
   margin-left: calc(50% - 50vw);
   margin-right: calc(50% - 50vw);
 }
+.az-video-background .bg-video-player-control {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  font-size: 0.9em;
+  color: #0c234b;
+  z-index: -1;
```

Video iframe set to -100
```suggestion
  z-index: -102;
```